installing.md

```diff
@@ -80,6 +80,13 @@ source /FULL/PATH/TO/wp-completion.bash
 
 To have this change take effect in your currently active shell, run `source ~/.bash_profile` afterwards.
 
+If you are using a Linux operating system, this process is slightly different. Instead of sourcing `wp-completion.bash` from your `~/.bash_profile`, you need to do so from your `~/.bashrc`. Make sure the `wp-completion.bash` file is in your home directory, then run the following commands from the home directory.
+
+```
+cat wp-completion.bash >> .bashrc
+source .bashrc
```

Comment:
Why not adding the same `source /FULL/PATH/TO/wp-completion.bash` line to `.bashrc` instead? Why `cat`?

Reply:
Shoot, this was so long ago now, that I'd have to check. I will see if I can spin up my Ubuntu boot this week to confirm. I _think_ it might have been due to the default configuration on Ubuntu, but I honestly can't remember.

Reply:
Ok. So this was a combination of the fact that a default Ubuntu install does not have a .bash_profile file, me not fully understanding what the different files like .bash_profile and .bashrc do, which I do now, and not realizing I could simply create the .bash_profile file and then follow the rest of the instructions.